stages/org.osbuild.oscap.remediation

```diff
@@ -35,7 +35,7 @@ def main(tree, options):
     for source in ("/dev", "/proc"):
```

Comment:
This mount and the symlink could maybe moved into a new `setup_env()` helper even? To make the testing simpler?

Reply:
Yeah still working on that change, I was trying to see if I could tidy up other parts of the `main` function too.
Should probably have opened this as a draft

Reply:
Maybe that can be it's own thing though

Reply:
Aha, sorry then! I looked too early - I will be patient :)

Reply:
Haha no sorry that was my fault :)